### PR TITLE
Milestone 2: To Do list: add & remove

### PR DESCRIPTION
**Pull request by Dangelo-JAN**
## Creating the functions to add and remove milestone 2: To Do list: add & remove

In this milestone we:

- Create the functions add, remove and set To Do

### CODE REVIEW

File: src/todolist.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import DeleteIcon from './delete_black_24dp.svg';
+
+const toDoContainer = document.getElementById('todo-container');
+export default class ToDoList {
+  createEmpty = () => {
+    const todoes = [];
+    localStorage.setItem('todoes', JSON.stringify(todoes));
+  };
+
```
Comment (keroloussamy) [inline src/todolist.js:9]: - [x] Kindly add a function for editing task descriptions, as the requirements say `Implement a function for editing task descriptions`.

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @Dangelo-JAN, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/todolist.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
+import DeleteIcon from './delete_black_24dp.svg';
+
+const toDoContainer = document.getElementById('todo-container');
+export default class ToDoList {
+  createEmpty = () => {
+    const todoes = [];
+    localStorage.setItem('todoes', JSON.stringify(todoes));
```
Comment (TedLivist) [inline src/todolist.js:7]: - [x] You are yet to implement the edit functionality as requested by the previous reviewer. Kindly fix that.

  The task description can be 'edited' but it does not persist in the localStorage after `blur` (leaving the task), on pressing the enter. Hence when I reload the task still remains the same.

  Tasks are added | Task is "edited" | Page is reloaded
  :---:|:---:|:---:
  dango-1 | dango-2 | dango-1

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Dangelo-JAN,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- [x] Kindly fix up your linters. Github Actions are working correctly now. Your linters are not working because you misspelled your directory name. You should have `.github/workflows` (with 's'), but you currently have `.github/workflow`.

  Github Actions | Microverse linter config | Your linter config
  :---:|:---:|:---:
  github-actions | microverse-linter | dango-linters

- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @Dangelo-JAN, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions:
- [ ] Kindly give the branch a meaningful name, not `feature-1` like the name of the project ` feature/add-remove `.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**